assets/blocks/sandstone.block

```diff
@@ -0,0 +1,9 @@
+/**
+ * This is just Sandstone from GenericRocks with a different texture
```

Comment:
Please just update the texture of the Sandstone in Generic Rocks.

For the future. Please consider everything in Generic Rocks and Soils as placeholder textures that can be replaced or modified to fit better into the world.